### DISCUSSION TITLE
problem updating the windows app SDK

### DISCUSSION THREAD

**Discussion by loreimal** [microsoft/WindowsAppSDK#4165] [Q&A]
Windows app SDK version 1.4.4
So I tried installing the windowsappruntimeinstall-x64.exe, and the cmd pops up for only half a second then disappears instantly, the update doesn't seem to be installed on my pc, not sure what is going on, any tips?

I also tried

1. installing previous versions with the same result
2. running as administrator
3. refreshing desktop
4. restarting PC
5. tried both preview and stable releases from [Microsoft's website](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/downloads)

**Comment by Yewzir** [comment]
@loreimal 

Try running the executable from Windows PowerShell or Command Prompt, and see what happens when you run it from there. Perhaps it will show you an informative message that you can actually read.

Kind regards
Yewzir
